src/main/java/org/example/BankingApp.java

```diff
@@ -0,0 +1,94 @@
+package org.example;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class BankingApp {
+
+
+    private static ArrayList<Account> accounts = new ArrayList<Account>();
+    private static final double INTEREST_RATE = 0.1;
+
+
+    public static void main(String[] args) {
+
+        BankingApp bankingApp = new BankingApp();
+        bankingApp.mainUI();
+    }
+    public void mainUI(){
+        /* This is the main UI */
+
+        Scanner scanner =new Scanner(System.in);
+
+        System.out.println("\n***************************************");
+        System.out.println("-------BANKING APP-------");
+        System.out.println("*********************************\n\n");
+
+        while (true){
+
+            System.out.println("Press 1 to Logging to App");
+            System.out.println("Press 2 to Create Account");
+            System.out.println("Press 3 to  Exit");
+
+            System.out.print("\nPlease enter your choice : ");
+            int enterChoice = scanner.nextInt();
+
+            if(enterChoice == 1){
+
+
```

Comment:
remove extra lines and call loggingUI function here

Reply:
done